### ISSUE TITLE
Deletions from Vault repository not handled properly

### ISSUE THREAD

**Issue by koverly** [slamotte/vault2git#12]
I've been testing with migrating a couple vault repositories and it appears the script does not handle cases in the Vault repository history where files were deleted.

I discovered this by running the script on a repository and then performing a diff between a fresh checkout from Vault and the new git repo.  Files that had been previously deleted in Vault were still present in the git repo.

I've never worked in ruby before but I will attempt to fix it and will submit a pull request if successful.

**Comment by koverly** [comment]
I've investigated this further and it appears to be a bug in SourceGear Vault.

I found [this thread](http://support.sourcegear.com/viewtopic.php?f=5&t=11145) on SourceGear's support forum.  The bug was reported in 2008 and acknowledged by SourceGear, however it has still not been fixed as of 2013 according to the thread.  That's really pathetic.

The work around is to delete the entire working folder prior to checking out another version from Vault.

The script contains the function clear_working_folder which deletes the contents of the working folder prior to checking out another version.  It did not seem to function with my first few tries.  The log output from the script would show  "DEBUG vault2git: Removing folders:" but the output showing the folders being deleted was empty.  Using a ruby interactive shell I was able to determine the line "files_to_delete = Dir[$options.dest + "/*"]" in clear_working_folder was returning an empty set.  The problem was that ruby expects forward slashes in the $options.dest path.  Passing the destination path using forward slashes worked.  The example path "c:\New\Git\Repo" in README.textile should be changed to "c:/New/Git/Repo".

For reference I'm using SourceGear Vault Professional 6.0.1 (30598).

Hope this helps someone else in the future.
